libraries/AP_Camera/AP_Camera_Backend.h

```diff
@@ -128,6 +132,12 @@ class AP_Camera_Backend
         uint32_t feedback_trigger_logged_count; // ID sequence number
     } camera_feedback;
 
+    // Picture settings
+    struct {
+        uint32_t time_interval;     // time interval (in miliseconds) between two consecutive pictures
```

Comment:
oh, sorry we missed this but if we're going to use milliseconds then time_interval should have "_ms" on the end.  So rename it to time_interval_ms.